src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -277,6 +278,25 @@
 
 		<template #footer>
 			<div class="left-sidebar__settings-button-container">
+				<NcButton v-if="supportsArchive && isFiltered === 'archived'"
+					type="tertiary"
+					wide
+					@click="handleFilter(null)">
+					<template #icon>
+						<IconArrowLeft :size="20" />
+					</template>
+					{{ t('spreed', 'Back to conversations') }}
+				</NcButton>
+				<NcButton v-else-if="supportsArchive"
+					type="tertiary"
+					wide
+					@click="handleFilter('archived')">
+					<template #icon>
+						<IconArchive :size="20" />
+					</template>
+					{{ t('spreed', 'Archived conversations') }}
+				</NcButton>
+
```

Comment:
I think it should not be seen when you don't have any archived conversations. 
P.S: also check if filter !== 'archived' in render condition so to not remove them when you are already on archived conversations page